packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -137,7 +137,8 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
 
   static defaultProps = {
     accessibility: alertBehavior,
-    dismissAction: { icon: 'close' },
+    // TODO: fix me
+    dismissAction: { icon: <CloseIcon /> },
```

Comment:
This is being fixes with https://github.com/microsoft/fluentui/pull/12531 (The changes here will be reverted as custom component is being added for the `dismissAction`)